.github/workflows/ui.yml

```
@@ -53,6 +53,7 @@ jobs:
           node-version: 16
           cache: pnpm
           cache-dependency-path: ui/pnpm-lock.yaml
+      - run: pnpm exec pip install -r ../requirements.txt
```

I think this should work, right?
```suggestion
      - run: pip install -r ../requirements.txt
```